drawbicyc/include/drawbicyc/plot_series.hpp

```diff
@@ -474,6 +496,24 @@ void plot_series(Plotter_t plotter, Plots plots, std::string type)
         }
         catch(...){;}
       }
+      else if (plt == "total_droplets_number")
+      {
+        // Total number of Cloud and Rain Droplets
+        try
+        {
+          // cloud fraction (cloudy if ql > 1e-5))
+          auto tmp = plotter.h5load_timestep("cloud_rw_mom0", at *n["outfreq"]);
+          typename Plotter_t::arr_t snap(tmp);
+          snap *= rhod * plotter.CellVol;
+          auto tmp2 = plotter.h5load_timestep("rain_rw_mom0", at *n["outfreq"]);
+          typename Plotter_t::arr_t snap2(tmp2);
+          snap2 *= rhod * plotter.CellVol;
+          auto tmp3 = plotter.h5load_timestep("all_rw_mom0", at *n["outfreq"]);
```

Comment:
all_rw_mom0 is in [1/kg]
To get the total number of droplets in a cell, you need to multiply by rhod and by cell volume.